static/src/stylesheets/module/_adslot.scss

```diff
@@ -60,6 +60,21 @@
     .ad-slot--sky & {
         width: $mpu-skyscraper-width;
     }
+
+    &--toggle {
+        margin: 0 auto;
+        @include mq($until: tablet) {
+            display: none;
+        }
```

Comment:
Do they always show beyond tablet?

Reply:
I believe so. I can't see any cases where we don't show beyond tablet in:

https://github.com/guardian/frontend/blob/aff2baffd7fdd26a70fd711a52ddf891d0f36037/static/src/stylesheets/module/_adslot.scss